lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -92,9 +92,36 @@ static bool isArrayLiteralIntrinsic(ApplyInst *ai) {
 }
 
 static ApplyInst *getAllocateUninitializedArrayIntrinsic(SILValue v) {
-  if (auto *applyInst = dyn_cast<ApplyInst>(v))
-    if (isArrayLiteralIntrinsic(applyInst))
-      return applyInst;
+  if (auto *ai = dyn_cast<ApplyInst>(v))
```

Comment:
I suggest braces around multiline bodies for clarity.

Reply:
The current style is to not add braces around nested `if`/`for` statements if the innermost block is one line long. I chose to adhere to that style here, since the lines are short. We could discuss changing the style later.